### PR TITLE
Hue: fix #3805

### LINKED ISSUE #3805: Hue binding: bad item type for channel type "switch"
https://github.com/eclipse/smarthome/blob/master/extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/thing/channels.xml#L35

The value for tag "item-type" is incorrectly set to `SwitchItem `rather than `Switch`.

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by sjka** [comment]
As it seems, the ip validation tool checks for the "committer" in git as well (which is some garbage when creating commits via Github's UI). The rest is okay though (author, signed-off) and I manually checked that the ECA is valid. The committer will be overridden anyway when squash-merging the PR, so it's safe to overrule the failed check.

File: extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/thing/channels.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
 	<!-- Switch Channel -->
     <channel-type id="switch">
-        <item-type>SwitchItem</item-type>
+        <item-type>Switch/item-type>
```
Comment (sjka) [inline extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/thing/channels.xml]: the `<` of the closing tag got lost

**Comment by lolodomo** [comment]
Very sorry for that, that is the risk when just editing in Git. That is now fixed.

**Comment by sjka** [comment]
Haha, no problem - it took me a few moments too to figure out what exactly made travis fail 😄

**Comment by sjka** [comment]
Thanks!

**Comment by sjka** [comment]
Thanks!

**Pull request merged**